src/mumble/LookConfig.cpp

```diff
@@ -109,6 +111,27 @@ QString LookConfig::title() const {
 	return tr("User Interface");
 }
 
+void LookConfig::on_qbBackgroundColor_clicked() {
+	QColor color = QColorDialog::getColor(Qt::white, this, tr("Choose a Color"));
+	if (color.isValid()) {
+		talkinguiBackgroundSet(color);
+	}
+}
+
+void LookConfig::talkinguiBackgroundSet(QColor color) {
+	selectedBackgroundColor = color;
+	QString style           = QString("background-color: %1;").arg(color.name());
+	qccolorPreview->setStyleSheet(style);
+	swBackgroundColor->setCurrentIndex(0);
+	qlBackgroundColor->setBuddy(qbClearBackgroundColor);
```

Comment:
Please set the ``buddy`` in the ``.ui`` file. Either by using QtDesigner, or by manually editing the ``.ui`` file. We don't want to set buddies at runtime